wagtail/images/blocks.py

```diff
@@ -51,3 +56,133 @@ def htmldiff(self):
                 "image_b": self.val_b,
             },
         )
+
+
+class ImageBlock(StructBlock):
```

Comment:
I think this is worth a docstring to explain to Wagtail users and maintainers that this StructBlock has magic in it so its `value` is an image model instance.